app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user
+
+  def current_user
+    @user ||= User.find(session[:user_id]) if session[:user_id]
+  end
 end
```

Comment:
Need to add sessions for real. This method is currently getting stubbed & not actually triggering the requirements to have a session.